package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "app",
+  "private": true,
+  "scripts": {
+    "build": "webpack --config webpack.config.js"
+  },
+  "devDependencies": {
+    "@babel/preset-env": "^7.23.9",
+    "babel-loader": "^9.1.3"
+  },
+  "dependencies": {
+    "@reduxjs/toolkit": "^2.2.1",
+    "axios": "^1.6.7",
+    "react-redux": "^9.1.0",
+    "redux": "^5.0.1"
+  }
```

Comment:
- [ ] When I precompile the app assets using this command `rails assets:precompile` the error in the image below is raised. Please fix that.
![image](https://github.com/faisalakb/Hello_world_OneApp/assets/68796232/bd56e1e9-d640-47a7-bceb-a32c544ad8e6)

This error happened because you forgot to install `webpack` and `webpack-cli` packages, Please install them and ensure the assets are precompiled successfully then kindly run the rails server and ensure that the app runs as expected as required by the previous reviewer and also to fulfill the project requirements.